GreysonElkins/abstractify
# Dependencies/ge setup

#### What's this PR do?
* Installs dependencies and introduces file tree
* Creates first Routes
* Sets up components (namely App, it's routes, and children)
#### Where should the reviewer start?
`App.js`
#### How should this be manually tested?
Go to the different Routes, see that "hello world" text references correct component
#### Any background context you want to provide?
I'm just trying to stay ahead of those messy dependency and set-up issues that'll pop up when/if the project gets big. 
#### What are the relevant tickets?
close #1, close #2, close #3, close #5, #8, #6, #7, #11, #12
#### Screenshots (if appropriate)
![pank](https://user-images.githubusercontent.com/62047446/93403947-24fa3d00-f846-11ea-8189-ee58ed35f33c.gif)
#### Questions:
I was surprised after installing `react-router` that I needed to install `react-router-dom`. Here's a question for my future self to find the answer to, "Why do I need both? (#2 )" -a younger lazier self.
